api_v3/services/BaseEntryService.php

```diff
@@ -1145,4 +1147,45 @@ public function restoreRecycledAction($entryId)
 		
 		return $this->getEntry($entryId);
 	}
+
```

Comment:
create a response object that contains the value - 
KalturaEntryPermission {
entryId - 
permissionLevel - <KalturaEntryPemissionLevel >
}